automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/FreeTrialSpec.scala

```diff
@@ -0,0 +1,76 @@
+package org.broadinstitute.dsde.firecloud.test.user
+
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.firecloud.api.{Orchestration, Thurloe}
+import org.broadinstitute.dsde.firecloud.auth.AuthToken
+import org.broadinstitute.dsde.firecloud.component.{Button, Label, TestId}
+import org.broadinstitute.dsde.firecloud.config.{Config, Credentials, UserPool}
+import org.broadinstitute.dsde.firecloud.fixture.UserFixtures
+import org.broadinstitute.dsde.firecloud.page.workspaces.WorkspaceListPage
+import org.broadinstitute.dsde.firecloud.test.{CleanUp, WebBrowserSpec}
+import org.scalatest.{BeforeAndAfter, FreeSpec, Matchers}
+
+import scala.util.Try
+
+
+/**
+  * Tests for new user registration scenarios.
+  */
+class FreeTrialSpec extends FreeSpec with BeforeAndAfter with Matchers with WebBrowserSpec
+  with UserFixtures with CleanUp with LazyLogging {
+
+  val adminUser: Credentials = UserPool.chooseAdmin
+  implicit val authToken: AuthToken = adminUser.makeAuthToken()
+
+  val testUser: Credentials = Config.Users.testUser  // TODO: pull from user pool and fetch correct subject ID
+  val userAuthToken: AuthToken = testUser.makeAuthToken()
+  var subjectId : String = Orchestration.profile.getUser()(userAuthToken)("userId").toString // TODO: Use orch endpoint to change trial status
+
+
+  // Clean-up anything left over from any previous failures.
+  before {
+    Try(Thurloe.keyValuePairs.delete(subjectId, "trialState"))
+  }
+
+  private def registerCleanUpForDeleteTrialState(subjectId: String): Unit = {
+    register cleanUp Thurloe.keyValuePairs.delete(subjectId, "trialState")
+  }
+
+  "FireCloud" - {
+
+    "should not show the free trial banner to a non-enabled user" in withWebDriver { implicit driver =>
+      withSignIn(testUser) { _ =>
+        await ready new WorkspaceListPage()
+        val bannerTitle = Label(TestId("trial-banner-title")).getText
+        bannerTitle shouldBe ""
```

Comment:
checking that the title is the empty string feels a little fragile - for instance, what if we are showing a banner but the underlying json in the bucket has an empty string? Could you test `Label(TestId("trial-banner-title")).isVisible` instead?